.travis.yml

```diff
@@ -11,8 +10,7 @@ before_script:
 
 install:
   - pip install --upgrade pip
-  - pip install -e .
-  - pip install slackclient  # Optional dependency but Slack tests are skipped without it
+  - pip install -e .[slack]
```

Comment:
I was thinking, maybe another simple test doing `pip install -e .[all]` might be useful to see if the entire dependency tree is still buildable.